MapboxCoreNavigation/NavigationEventsManager.swift

```diff
@@ -107,18 +109,34 @@ open class NavigationEventsManager: NSObject {
         return event
     }
     
-    func navigationDepartEvent() -> EventDetails? {
+    func navigationRouteRetrievalEvent() -> PerformanceEventDetails? {
+        guard let sessionState = sessionState,
+            let responseEndDate = sessionState.currentRoute.responseEndDate,
+            let fetchStartDate = sessionState.currentRoute.fetchStartDate else {
+            return nil
+        }
+
+        var event = PerformanceEventDetails(event: NavigationEventTypeRouteRetrieval, session: sessionState, createdOn: sessionState.currentRoute.responseEndDate)
+        event.counters.append(PerformanceEventDetails.Counter(name: "elapsed_time",
+                                                              value: responseEndDate.timeIntervalSince(fetchStartDate)))
```

Comment:
This is measured in seconds (as a floating-point number), but mapbox/mapbox-navigation-android#1661 implemented the field in nanoseconds as a long integer.

Reply:
mapbox/mapbox-navigation-android#1732